New version: MATLAB v0.9.0

JuliaRegistrator opened the pull request:
- Registering package: MATLAB
- Repository: https://github.com/JuliaInterop/MATLAB.jl
- Created by: @tqml
- Version: v0.9.0
- Commit: d069345e86022c907b7a2786ac1f9b4c6ab23d73
- Reviewed by: @tqml
- Reference: https://github.com/JuliaInterop/MATLAB.jl/commit/d069345e86022c907b7a2786ac1f9b4c6ab23d73#commitcomment-153522026
- Description: Calling MATLAB in Julia through MATLAB Engine
- Release notes:
<!-- BEGIN RELEASE NOTES -->
`````
## Breaking changes

- Types that are subtypes of AbstractArray (e.g. Diagonal) are now converted to MATLAB arrays instead of structs. Within MATLAB these types are represented as normal, _dense_ arrays. [`#229`](https://github.com/JuliaInterop/MATLAB.jl/pull/229)
- Tuples are no longer converted to MATLAB dictionaries. Instead tuples are converted to cell arrays (if it has mixed types) or numeric arrays (if all elements share a single numeric type) [`#231`](https://github.com/JuliaInterop/MATLAB.jl/pull/231)
`````
<!-- END RELEASE NOTES -->

<!-- bf0c69308befbd3ccf2cc956ac8a46712550b79fc9bfb5e4edf8f833f05f4c18b06eddad8845b45beb9f45c2b8020dd6c27d2531c9b91adac945321beb023722b055491f5cecd068684ead2f3ff6d9eb2276e0e9f1238525ad8c48e2cc0fd0e89322c3bb18bcb5ce6061bd4ddaa971f8fcdc0c9e39f700b258946400b6e985a926b30fe2efe4a3f37688a91bae99b7382be86844ec54eb30dca40c64cac2d74870293b6d922ad623939da33cad87cad3297f109b3b4447250550861aa662e2553c8de0efe378bbc355b94bb67ad915d45817d820e7041d20614e63c47132d49c -->

giordano commented:
@musm please see the message above.  Once you fix the automerge issues you can register again the new revision without changing the version number and this pull request will be automatically updated.

[noblock]

giordano commented:
1. Read very carefully https://github.com/JuliaRegistries/General/pull/124734#issuecomment-2648871011, in particular the section "1. [AutoMerge Guidelines](https://juliaregistries.github.io/RegistryCI.jl/stable/guidelines/) which are not met ❌", including the section giving an example of how to add release notes
2. You're blocking registration with your comment missing `[noblock]` (as also explained by the message by the bot that I linked above)

kyak commented:
@musm @tqml please provide information requested so that this can be merged. Thank you!

JuliaRegistrator committed:
New version: MATLAB v0.9.0

UUID: 10e44e05-a98a-55b3-a45b-ba969058deb6
Repo: https://github.com/JuliaInterop/MATLAB.jl.git
Tree: 005c36caa0ad24855c3292020c01639b728e5081

Registrator tree SHA: 17aec322677d9b81cdd6b9b9236b09a3f1374c6a

tqml commented:
Hope this works now :) 

[noblock]

tqml commented:
Ok, updated. 

[noblock]

tqml commented:
Nice, it worked. Thanks for the help @giordano :) 
[noblock]
